quora-api/src/main/resources/application.yaml

```diff
@@ -12,7 +12,7 @@ spring:
     driverClassName: org.postgresql.Driver
     url: jdbc:postgresql://localhost:5432/quora
     username: postgres
-    password: password
+    password: admin

```

Comment:
Can we all have a password set as  "password" only.

Reply:
Sure